accepted/global-json-updates.md

```diff
@@ -0,0 +1,790 @@
+| 2.1.500                                              | fail      | fail    | fail    | fail    | fail        | fail          | fail        | fail        | fail    |
+| 2.1.501, 2.1.503                                     | _2.1.501_ | 2.1.501 | 2.1.501 | 2.1.501 | _2.1.503_   | 2.1.503       | 2.1.503     | 2.1.503     | 2.1.501 |
+| 2.1.503, 2.1.505, 2.1.601, 2.2.101, 3.0.100          | 2.1.503   | 2.1.503 | 2.1.503 | 2.1.503 | 2.1.505     | 2.1.601       | 2.2.101     | 3.0.100     | fail    |
+| 2.1.601, 2.1.604, 2.1.702, 2.2.101, 2.2.203, 3.0.100 | fail      | 2.1.601 | 2.1.601 | 2.1.601 | fail        | 2.1.702       | 2.2.203     | 3.0.100     | fail    |
+| 2.2.101, 2.2.203, 3.0.100                            | fail      | fail    | 2.2.101 | 2.2.101 | fail        | fail          | 2.2.203     | 3.0.100     | fail    |
+| 3.0.100, 3.1.102                                     | fail      | fail    | fail    | 3.0.100 | fail        | fail          | fail        | 3.1.102     | fail    |
+
+Or, if it is easier to understand with logic (bail out of logic on fail or select):
+
+1. If not `latest`, and there is a match, select it.
+1. If we get here for `disable`, then fail.
+1. Exclude any lower versions.
+1. If `minor` or `latestMinor`, exclude any that don't match major version. 
+1. If `feature` or `latestFeature`, exclude any that don't match major and minor version.
+1. If `patch` or `latestPatch`, exclude any that don't match major, minor and feature band version.
+1. If there is one left, select it.
+1. If any of the `latest` options are used, select the highest. 
+1. Otherwise, select the lowest major, and within that lowest minor, and within that lowest feature band. Then select the highest patch in that feature band.  
+
+## Customer problems addressed
+
+* User wants to install a preview but only use it with a subset of projects.
+  * User opts out in projects where preview should not be used by adding `ignorePreview: true` to those projects, or
+  * User adds `ignorePreview: true` in a `global.json` higher in their file system, and adds `ignorePreview: false` in a `global.json` in specific projects.
+* User wants to always use a specific SDK patch version, and receive an error if it is not on the machine.
+  * User adds `rollForward: disable` to a `global.json`
+* User wants to always run the latest patch of a particular SDK feature band.
+  * User adds `rollForward: latestPatch` and the lowest version (with patch) they want to a `global.json`
+* User includes a feature that requires an SDK that corresponds to a SDK feature band or higher, and wants to run the best available tools.
+  * User adds `rollForward: latestMajor` and the lowest version (with patch) they want to a `global.json`
+* User wants to know what version of the SDK is being used.
+  * Not addressed in this proposal: `global.json` will still be a subtle, hidden and easily forgotten marker
```

Comment:
It is odd to see multiple instances of "not address in this proposal" in a section called "Customer problems addressed".